pittapi/library.py

```diff
@@ -19,7 +19,7 @@
 
 import requests
 from html.parser import HTMLParser
-from typing import Any, Dict, List
+from typing import Any, Dict, List, Union
```

Comment:
FYI, Python has built-in type hints for the most common types such as `list`, `dict`, etc., which means that you don't need most of these imports from the `typing` library. For example, instead of
```py
from typing import Dict, List, Union

...

def reserved_hillman_times() -> List[Dict[str, Union[str, List[str]]]]:
```
you can now write
```py
...

def reserved_hillman_times() -> list[dict[str, str | list[str]]]:
```
I'm not gonna ask you to change all the type hints to not use the `typing` library (since this file was already using it), but it's good to know that these types are now a built-in feature.

Reply:
Alright. I didn't realize this was natively supported. Good to know for future reference.